src/elements/NonlinearLens.H

```diff
@@ -19,9 +19,10 @@
 #include "mixin/nofinalize.H"
 
 #include <AMReX_Extension.H>
+#include <AMReX_GpuComplex.H>
 #include <AMReX_Math.H>
 #include <AMReX_REAL.H>
-#include <AMReX_GpuComplex.H>
+// TODO: #include <AMReX_SIMD.H>
```

Comment:
Note: There is a TODO here again in line 40.

Reply:
Yes, I cannot implement this yet because `Complex` math is not yet fully supported. This is something coming in C++26 and might earlier come to vir-simd.

I will add an inline comment linking https://github.com/mattkretz/vir-simd/issues/42

Reply:
Done.